### ISSUE TITLE
//src/test/shell/bazel:bazel_apple_test is failing on Darwin

### ISSUE THREAD

**Issue by meteorcloudy** [bazelbuild/bazel#1828]
http://ci.bazel.io/job/bazel-tests/BAZEL_VERSION=HEAD,PLATFORM_NAME=darwin-x86_64/239/consoleFull
```
ERROR: missing input file '@bazel_tools//tools/objc:make_hashed_objlist.py'.
ERROR: /private/var/tmp/_bazel_ci/e5bd876f99600cb31ceb051a9d639b5b/execroot/darwin-x86_64/_tmp/bazel_apple_test_88/root/8ff5724fc94fa394ad9c1677f94eae17/external/bazel_tools/tools/objc/BUILD:66:1: @bazel_tools//tools/objc:libtool: missing input file '@bazel_tools//tools/objc:make_hashed_objlist.py'.
____Building complete.
Target //ios:app failed to build
ERROR: /private/var/tmp/_bazel_ci/e5bd876f99600cb31ceb051a9d639b5b/execroot/darwin-x86_64/_tmp/bazel_apple_test_88/root/8ff5724fc94fa394ad9c1677f94eae17/external/bazel_tools/tools/objc/BUILD:66:1 1 input file(s) do not exist.
____Elapsed time: 22.023s, Critical Path: 0.40s
------------------------------------------------------------------------
test_build_app FAILED: should pass .
/private/var/tmp/_bazel_ci/e5bd876f99600cb31ceb051a9d639b5b/execroot/darwin-x86_64/bazel-out/local-fastbuild/bin/src/test/shell/bazel/bazel_apple_test.runfiles/io_bazel/src/test/shell/bazel/bazel_apple_test:131: in call to test_build_app
FAILED: test_build_app
```

**Comment by meteorcloudy** [comment]
//cc @dmishe Can you take a look?

**Comment by meteorcloudy** [comment]
//src/test/shell/bazel:bazel_example_test is also failing due to same reason

```
(cd /private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_example_test_1/root/0cbc8fcb18d937649cafbe31aa05943c/execroot/workspace && \
  exec env - \
    PATH=.:/Users/ci/workspace/bazel-tests/BAZEL_VERSION/HEAD/PLATFORM_NAME/darwin-x86_64/.bin:/usr/bin:/bin:/usr/sbin:/sbin \
    TMPDIR=/var/folders/3b/9pdj48r577v5qqr3dqn58xq40000gn/T/ \
  /private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_example_test_1/root/0cbc8fcb18d937649cafbe31aa05943c/execroot/workspace/_bin/build-runfiles bazel-out/host/bin/external/bazel_tools/tools/objc/xcrunwrapper.runfiles_manifest bazel-out/host/bin/external/bazel_tools/tools/objc/xcrunwrapper.runfiles)
ERROR: missing input file '@bazel_tools//tools/objc:make_hashed_objlist.py'.
ERROR: /private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_example_test_1/root/0cbc8fcb18d937649cafbe31aa05943c/external/bazel_tools/tools/objc/BUILD:102:1: Creating runfiles tree bazel-out/host/bin/external/bazel_tools/tools/objc/xcrunwrapper.runfiles [for host] failed: build-runfiles failed: error executing command 
  (cd /private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_example_test_1/root/0cbc8fcb18d937649cafbe31aa05943c/execroot/workspace && \
  exec env - \
    PATH=.:/Users/ci/workspace/bazel-tests/BAZEL_VERSION/HEAD/PLATFORM_NAME/darwin-x86_64/.bin:/usr/bin:/bin:/usr/sbin:/sbin \
    TMPDIR=/var/folders/3b/9pdj48r577v5qqr3dqn58xq40000gn/T/ \
  /private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_example_test_1/root/0cbc8fcb18d937649cafbe31aa05943c/execroot/workspace/_bin/build-runfiles bazel-out/host/bin/external/bazel_tools/tools/objc/xcrunwrapper.runfiles_manifest bazel-out/host/bin/external/bazel_tools/tools/objc/xcrunwrapper.runfiles): com.google.devtools.build.lib.shell.AbnormalTerminationException: Process terminated by signal 15.
ERROR: /private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_example_test_1/root/0cbc8fcb18d937649cafbe31aa05943c/external/bazel_tools/tools/objc/BUILD:66:1: @bazel_tools//tools/objc:libtool: missing input file '@bazel_tools//tools/objc:make_hashed_objlist.py'.
____Building complete.
Target //examples/objc:PrenotCalculator failed to build
ERROR: /private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_example_test_1/root/0cbc8fcb18d937649cafbe31aa05943c/external/bazel_tools/tools/objc/BUILD:66:1 1 input file(s) do not exist.
____Elapsed time: 31.134s, Critical Path: 2.04s
-- Test log: -----------------------------------------------------------
------------------------------------------------------------------------
test_objc FAILED: Failed to build --ios_sdk_version=9.3 //examples/objc:PrenotCalculator .
/private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/bazel-out/local-fastbuild/bin/src/test/shell/bazel/bazel_example_test.runfiles/io_bazel/src/test/shell/bazel/test-setup.sh:223: in call to assert_build
/private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/bazel-out/local-fastbuild/bin/src/test/shell/bazel/bazel_example_test.runfiles/io_bazel/src/test/shell/bazel/test-setup.sh:229: in call to assert_build_output
/private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/bazel-out/local-fastbuild/bin/src/test/shell/bazel/bazel_example_test.runfiles/io_bazel/src/test/shell/bazel/bazel_example_test:105: in call to test_objc
FAILED: test_objc
```

**Comment by meteorcloudy** [comment]
Same reason for `//src/test/shell/bazel:bazel_objc_test`
```
** test_archive_timestamps *****************************************************
-- Test log: -----------------------------------------------------------
INFO: Reading 'startup' options from /private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_objc_test_1/bazelrc: --output_user_root=/private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_objc_test_1/root --host_javabase=/private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/bazel-out/local-fastbuild/bin/src/test/shell/bazel/bazel_objc_test.runfiles/local_jdk
INFO: $TEST_TMPDIR defined: output root default is '/private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_objc_test_1'.
____Loading package: objclib
____Loading...
WARNING: /private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_objc_test_1/root/95d7d7c2a5f598d3f3db1ac0be525969/external/bazel_tools/tools/osx/xcode_configure.bzl:57:5: Invoking xcodebuild failed, return code 256, stderr: Process terminated by signal 15.
____Loading complete.  Analyzing...
____Loading package: @bazel_tools//tools/objc
____Loading package: @bazel_tools//third_party/java/jdk/langtools
____Found 1 target...
____Building...
____[0 / 11] BazelWorkspaceStatusAction stable-status.txt
____[3 / 11] Creating runfiles tree bazel-out/host/bin/external/bazel_tools/tools/objc/libtool.runfiles [for host]
____[6 / 11] Symlinking @bazel_tools//tools/objc:libtool [for host]
____[7 / 12] Writing file external/bazel_tools/tools/objc/xcodegen.jar-2.params [for host]
____[8 / 13] Symlinking @bazel_tools//tools/objc:xcrunwrapper [for host]
ERROR: missing input file '@bazel_tools//tools/objc:make_hashed_objlist.py'.
ERROR: /private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_objc_test_1/root/95d7d7c2a5f598d3f3db1ac0be525969/external/bazel_tools/tools/objc/BUILD:66:1: @bazel_tools//tools/objc:libtool: missing input file '@bazel_tools//tools/objc:make_hashed_objlist.py'.
____Building complete.
Target //objclib:objclib failed to build
ERROR: /private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/_tmp/bazel_objc_test_1/root/95d7d7c2a5f598d3f3db1ac0be525969/external/bazel_tools/tools/objc/BUILD:66:1 1 input file(s) do not exist.
____Elapsed time: 10.284s, Critical Path: 1.11s
------------------------------------------------------------------------
test_archive_timestamps FAILED: Should build objc_library .
/private/var/tmp/_bazel_ci/663aef0b024d9552f81e09f650e1ea55/execroot/darwin-x86_64/bazel-out/local-fastbuild/bin/src/test/shell/bazel/bazel_objc_test.runfiles/io_bazel/src/test/shell/bazel/bazel_objc_test:203: in call to test_archive_timestamps
FAILED: test_archive_timestamps
```

**Comment by dmishe** [comment]
Sure, it looks like we don't have the required dep make_hashed_objlist.py packaged.

**Comment by dmishe** [comment]
Have to send this back to @c-parsons as the dependencies looks correct to me and there are no obvious reasons why the test sandbox isn't picking those up.

**Comment by katre** [comment]
Fixed in bebbbe5d30f2a05e93780306208e1933d664d71b.
